### PR TITLE
updating for prod deploy

### PR DESCRIPTION
**Pull request by sandy41**

### CODE REVIEW

File: cinder-upgrade/update-apt-source-file.yaml [hunk 1]
```diff
@@ -3,12 +3,9 @@
...
-      lineinfile: dest=/etc/apt/sources.list line="{{cinder_old_repo_path}}" state=absent
-      become: yes
-    - name: Remove old cinder packages path
-      lineinfile: dest=/etc/apt/sources.list line="# {{cinder_old_repo_path}}"
-      become: yes
     - name: Add cinder packages path
-      lineinfile: dest=/etc/apt/sources.list line="{{cinder_repo_path}}"
+      lineinfile: dest=/etc/apt/sources.list regexp='^deb.*10.140.221.229\/snapshots.*' line="{{cinder_repo_path}}" backup=yes
+      become: yes
+    - name: Update rados_connect_timeout in cinder.conf
```
Comment (souvik-ray) [inline cinder-upgrade/update-apt-source-file.yaml]: I d rather change this (time- outs) in the sbs-config file and redeploy the package....
Reason: We should always have the exact copy of configs in the repository.....

I am fine with this for today's deployment but we should add this line .... sbs-config

File: cinder-upgrade/deploy-cinder-backup-hosts.yaml [hunk 2]
```diff
@@ -1,13 +1,10 @@
-- hosts: hostList
+- hosts: backupList
```
Comment (souvik-ray) [inline cinder-upgrade/deploy-cinder-backup-hosts.yaml]: This change should be refected in VARS .. Let me know if it is already done...

File: cinder-upgrade/deploy-cinder-volume-hosts.yaml [hunk 3]
```diff
@@ -1,13 +1,10 @@
-- hosts: hostList
+- hosts: volList
```
Comment (souvik-ray) [inline cinder-upgrade/deploy-cinder-volume-hosts.yaml]: This change should be refected in VARS .. Let me know if it is already done...

File: cinder-upgrade/update-apt-source-file.yaml (hunk 1 above)
Comment (vivekdhayaal) [inline cinder-upgrade/update-apt-source-file.yaml]: Already added in sbs-config long back...
https://github.com/JioCloud/sbs-config/pull/29

File: cinder-upgrade/deploy-cinder-volume-hosts.yaml (hunk 3 above)
Comment (souvik-ray) [inline cinder-upgrade/deploy-cinder-volume-hosts.yaml]: Ditto... Ignore..

**Pull request merged**